tests/v2/validation/terratest/cluster/build_module_test.go

```diff
@@ -0,0 +1,31 @@
+package terratest
+
+import (
+	"testing"
+
+	cleanup "github.com/rancher/rancher/tests/terratest/functions/cleanup"
+	terratest "github.com/rancher/rancher/tests/terratest/functions/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type BuildModuleTestSuite struct {
+	suite.Suite
+}
+
+func (r *BuildModuleTestSuite) TestBuildModule() {
+	r.T().Parallel()
+
+	// Clean main.tf
```

Comment:
Comments here can be removed.

Reply:
Is the team consensus to remove these comments?